scripts/coverage.sh

```diff
@@ -12,7 +12,7 @@ cores=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu)
 make hyriseTest -j $((cores / 2))
 cd -
 
-./$1/hyriseTest
+./$1/hyriseTest $3
```

Comment:
This will break if there is no second parameter. The call structure is not optimal, anyway (see above).